frontend/src/index.css

```
@@ -252,6 +252,41 @@ body {
+    mask-image: linear-gradient(
+      to bottom,
+      black 0,
+      black calc(100% - 2rem),
+      transparent 100%
+    );
+  }
+
+  .shimmer {
+    background: linear-gradient(
+      90deg,
+      currentColor 35%,
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)